apps/callscreen/js/calls_handler.js

```diff
@@ -277,45 +277,47 @@ var CallsHandler = (function callsHandler() {
   }
 
   function handleCallWaiting(call) {
-    LazyL10n.get(function localized(_) {
-      var number = call.secondId ? call.secondId.number : call.id.number;
+    var number = call.secondId ? call.secondId.number : call.id.number;
+
+    if (!number) {
+      navigator.mozL10n.setAttributes(CallScreen.incomingNumber,
+                                      'withheld-number');
+      // Force a synchronous translation so we can resize the string
+      navigator.mozL10n.translateFragment(CallScreen.incomingNumber);
```

Comment:
this is risky because translateFragment may not be sync (and in API 3 it is not sync) https://github.com/stasm/l20n.js/blob/lightweight/src/bindings/html/dom.js#L36

While in API3 you can use the returned promise, it will still only resize once. If we retranslate later, you will not resize.

If you need to resize when the value changes, why not set up MutationObserver on the element?